scripts/sct_warp_template.py

```diff
@@ -88,13 +88,14 @@ def __init__(self, fname_src, fname_transfo, warp_atlas, warp_spinal_levels, fol
             sct.printv('\nWARP SPINAL LEVELS:', self.verbose)
             warp_label(self.path_template, self.folder_spinal_levels, param.file_info_label, self.fname_src, self.fname_transfo, self.folder_out)
 
-        # to view results
-        sct.printv('\nDone! To view results, type:', self.verbose)
-        sct.printv('fslview ' + self.fname_src + ' ' \
-                   + self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'T2') + ' -b 0,4000 ' \
-                   + self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'vertebral') + ' -l MGH-Cortical -t 0.5 ' \
-                   + self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'gray matter') + ' -l Red-Yellow -b 0.5,1 ' \
-                   + self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'white matter') + ' -l Blue-Lightblue -b 0.5,1 &\n', self.verbose, 'info')
+        # display viewer syntax
+        sct.display_viewer([self.fname_src,
+                            self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'T2'),
+                            self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'gray matter'),
+                            self.folder_out + self.folder_template + get_file_label(self.folder_out + self.folder_template, 'white matter')],
+                           l_colormap=['gray', 'gray', 'red-yellow', 'blue-lightblue'],
+                           l_opacity=['1', '1', '0.5', '0.5'],
+                           verbose=param.verbose)
```

Comment:
Maybe this is an example to better appreciate using a list of structures:
```python
sct.display_viewer(
 dict(
  filename=self.fname_src,
  colormap="gray",
  opacity=1,
 ),
 dict(
  filename=...,
 ),
)
```

Reply:
one disadvantage i see (unless there is a workaround to it), is cases where the number of elements in the output list is unknown:
https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_analyze_texture.py#L343

however, i think this is the only example in SCT, so we can find a workaround specifically for this function, and use list of structures everywhere else.